nodes/text/debug_print.py

```diff
@@ -61,7 +61,7 @@ def process(self):
             return
         for i, socket in enumerate(self.inputs):
             if socket.is_linked and self.print_socket[i]:
-                print(socket.sv_get(deepcopy=False))
+                self.debug(socket.sv_get(deepcopy=False))
```

Comment:
i'm pretty sure this node should always print its data, regardless of logging level?

Reply:
But, it is called "debug print"... :) I'm not sure. I can change this to self.info instead, for example, to make output visible by default. Or, I can revert this back to print, but then the output will not be visible in logs, only in console.

Reply:
the node is meant to output the content of the incoming socket into console, that's what I use it for sometimes. so it would be a bit of pain to have to enable an option in preferences for it to work..and disable it when your done..

Reply:
It is also possible to set logging level to DEBUG for this one line and then set it back. Not sure if it is a good idea.

Reply:
`self.info()` makes a bit more sense in this specific instance.